GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml

```diff
@@ -19,22 +19,44 @@
               <td class="change-answer"><a href="@Url.Action("Part", "OrganisationDetails", new { id = 2 })">Change<span class="visually-hidden"> @Html.LabelFor(m => m.BusinessName.BusinessName)</span></a></td>
             }
         </tr>
+        
         <tr>
             <td>@Html.LabelFor(m => m.BusinessName.TradingName)</td>
-            <td class="@Model.BusinessName?.TradingName">@Html.DisplayFor(m => m.BusinessName.TradingName, new { @class = "org" })</td>
+            <td class="@Model.BusinessName?.TradingName">
+            @if (Model.BusinessName.HasTradingName ?? false)
+            {
+                @Html.DisplayFor(m => m.BusinessName.TradingName, new { @class = "org" })
+            }
+            else
+            {
+                var text = "Not set";
```

Comment:
Do we need this variable?

Reply:
I think we do unless it's html wrapped, now I think about it I could have done that in the first place! missed it!

Reply:
Oh yeah, I see what you mean.